help/destinations/catalog/personalization/pega.md

```diff
@@ -1,13 +1,17 @@
 ---
-title: Pega Customer Decision Hub connection
-description: Use the Pega Customer Decision Hub destination in Adobe Experience Platform to send profile attributes and audience membership data to Pega Customer Decision Hub for next-best-action decisioning.
+title: (V1) Pega CDH Realtime Audience connection
+description: Use the Pega Customer Decision Hub Realtime Audience destination in Adobe Experience Platform to send profile attributes and audience membership data to Pega Customer Decision Hub for next-best-action decisioning.
 exl-id: 0546da5d-d50d-43ec-bbc2-9468a7db4d90
 ---
-# Pega Customer Decision Hub connection
+# Pega CDH Realtime Audience connection
+
+>[!IMPORTANT]
+>
+>This version of the Pega Customer Decision Hub Realtime Audience destination supports only a single Pega Customer Decision application. If you have multiple Pega Customer Decision Hub applications configured, you need to use the [(V2) Pega CDH Realtime Audience destination connector](https://github.com/paratAtPega/experience-platform.en/edit/create-destination/help/destinations/catalog/personalization/pega-v2.md).
```

Comment:
That's not the correct way to link to the new page. Use the standard markdown linking. `(V2) Pega CDH Realtime Audience destination connector](pega-v2.md)`

Reply:
updated